package.json

```diff
@@ -1,8 +1,10 @@
 {
   "name": "tweakpane-plugin-thumbnail-list",
-  "version": "0.3.0",
+  "version": "1.0.0",
```

Comment:
Nit: Let's keep this at v0.3.0 within the PR, and I'll bump to the next major version when cutting the release.